framework/components/ecl-tabs/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@ec-europa/ecl-navigation-lists",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "0.1.0",
+  "description": "ECL Tabs",
+  "main": "_ecl-tabs.scss",
+  "style": "_ecl-tabs.scss",
+  "peerDependencies": {
+    "@ec-europa/ecl-links": "^0.1.0",
```

Comment:
At some point, we will have to rethink the way we handle the dependencies between the packages. Right now, we only use `peerDependencies` which are more informational / not strict enough. I guess that we will need to switch to `dependencies`, but it will also slow the bootstrap phase since lerna will have to link all the dependencies between them correctly...